### PR TITLE
dockerize the project

### PR DESCRIPTION
**Pull request by thedarkrai07**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+.github/
+.husky/
```
Comment (jordaniza) [inline .dockerignore:3]: May want to exclude coverage, logs

Comment (thedarkrai07) [reply .dockerignore]: fixed

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      - mongo
+
+  react-app:
+    build: "."
+    ports:
+      - 3000:3000
+    stdin_open: true
+    tty: true
+    volumes:
+      - ./packages:/bountyboard/packages
```
Comment (jordaniza) [inline docker-compose.yml:27]: I'd probably build only the react app in the react app container

**Review by jordaniza — DISMISSED** [review]
Per discussions on discord and confirmation of integration tests passing - approved.

**Review by jordaniza — APPROVED** [review]

**Pull request merged**